internal/log/log.go

```diff
@@ -60,6 +60,19 @@ const (
 	spinnerText        = " waiting"
 )
 
+type SettingsInterface interface {
```

Comment:
I think we should name it just `Settings`

Reply:
Let's move this interface to the `settings.go`?

Reply:
done